lib/routes/gmu/news.ts

```diff
@@ -0,0 +1,136 @@
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import type { Context } from 'hono';
+import type { Route } from '@/types';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/news/:type',
+    categories: ['university'],
+    example: '/gmu/news/gyyw',
+    parameters: {
+        type: {
+            description: '新闻类型，见下表，默认为 gyyw',
+            options: [
+                { value: 'gyyw', label: '赣医要闻' },
+                { value: 'ybdt', label: '院部动态' },
+                { value: 'mtgy', label: '媒体赣医' },
+                { value: 'xsjz', label: '学术讲座' },
+            ],
+            default: 'gyyw',
+        },
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['gmu.cn/xwzx/gyyw.htm', 'gmu.cn/'],
+            target: '/news/gyyw',
+        },
+        {
+            source: ['gmu.cn/xwzx/ybdt.htm'],
+            target: '/news/ybdt',
+        },
+        {
+            source: ['gmu.cn/xwzx/mtgy.htm'],
+            target: '/news/mtgy',
+        },
+        {
+            source: ['gmu.cn/xwzx/xsjz.htm'],
+            target: '/news/xsjz',
+        },
+    ],
+    name: '新闻中心',
+    maintainers: ['FrankFahey'],
+    url: 'gmu.cn/xwzx/gyyw.htm',
+    handler,
+};
+
+export async function handler(ctx: Context) {
+    const type = (ctx.req.param?.('type') as string) || 'gyyw';
```

Comment:
https://github.com/DIYgod/RSSHub/blob/9b62e77afb93fbee1d856d52940a7043c1933ea7/lib/routes/gmu/news.ts#L9 define `type` as compulsory path parameter which means `type` is not expected to have any default value.